### PR TITLE
Feat room page

### PR DESCRIPTION
**Pull request by KirillSerg**
image

### CODE REVIEW

**Comment by alexandrtovmach** [comment]
1. align vertically players name and red cross with radio button
2. use checkbox instead of radio button as it requested in task
3. move both ready checkbox and red cross to right side from username and stretch it to full width
4. room name -> should be input, not text
5. game name -> should be select input, not ordered list
6. align everything by left side, and general tip: NEVER USE CENTRAL ALIGNMENT (did you completed reading book "design for non-designers"?)

**Review by alexandrtovmach — CHANGES_REQUESTED** [review]
1. align vertically players name and red cross with radio button
2. use checkbox instead of radio button as it requested in task
3. move both ready checkbox and red cross to right side from username and stretch it to full width
4. room name -> should be input, not text
5. game name -> should be select input, not ordered list
6. align everything by left side, and general tip: NEVER USE CENTRAL ALIGNMENT (did you completed reading book "design for non-designers"?)

**Comment by KirillSerg** [comment]
image

File: src/wired.d.ts [hunk 1]
```diff
@@ -1,27 +1,27 @@
 declare namespace JSX {
```
Comment (alexandrtovmach) [inline src/wired.d.ts]: why did you touch this file? please revert any changes here

**Review by alexandrtovmach — CHANGES_REQUESTED** [review]

File: src/wired.d.ts (hunk 1 above)
Comment (KirillSerg) [reply src/wired.d.ts]: Without this change I can't add attributes like "checked, disabled..." in to the wired-elements. Typescript give me an issue, and I haven't another solution. I did the change in two elements anb add one element "wired-item"

Comment (KirillSerg) [reply src/wired.d.ts]: done

**Review by alexandrtovmach — APPROVED** [review]

**Pull request merged**